### DISCUSSION TITLE
Does Bluetooth 5.0 BLE LR boost range in advertise mode?

### DISCUSSION THREAD

**Discussion by evlo** [pvvx/ATC_MiThermometer#384] [Q&A] [answered]
I tried enabling Long range mode on LYWSD03MMC and see if I can catch signal on the longer distance, in _nrf connect_ android application I could connect to it pon PHY long range, but in advertisement i did pick up device on generally same distance as normal BLE 4.2 devices, maybe tiny bit further, but not even 100meters + trough the 60cm thick wall.

I did not do the line of sight test as that is not a use case for these kind of sensors (in my opinion).

Does LR even do anything in advertise mode?

I also did left RX power untouched.

**Comment by pvvx** [comment, marked as answer]
There is a big dependence on the chip used in the receiver.
Modern chips have a reception level of -110 dBm in LE Long Range mode.
For example RTL8761BU:
![image](https://github.com/pvvx/ATC_MiThermometer/assets/12629515/fee06e1a-2b15-4ad5-aae5-42223f042d1a)

**Reply by pvvx** [reply]
> I did not do the line of sight test as that is not a use case for these kind of sensors (in my opinion).

Serving personal areas requires communication range beyond Wi-Fi or Zigbee. In this case, the LE Long Range is suitable.
I do not have enough communication range for distances of 200 meters between my houses and buildings. Long range LE only.

**Reply by KeinPlandavonviel** [reply]
I hope it's ok if I just ask here. Which chip can you recommend for long range in combination with LYWSD03MMC? Then I would look for suitable adapters.
